cirq/sim/density_matrix_simulator.py

```diff
@@ -120,14 +119,36 @@ def __init__(self,
                  *,
                  dtype: Type[np.number] = np.complex64,
                  noise: 'cirq.NOISE_MODEL_LIKE' = None,
-                 seed: value.RANDOM_STATE_LIKE = None):
+                 seed: value.RANDOM_STATE_LIKE = None,
+                 replace_measurement_with_dephasing: bool = False):
```

Comment:
Rename this to `ignore_measurement_results`, as more general kinds of measurement will not be equivalent to dephasing.

Reply:
Done